src/TLuaInterpreter.cpp

```diff
@@ -3224,6 +3224,12 @@ int TLuaInterpreter::setFont(lua_State* L)
     }
     QString font{lua_tostring(L, s)};
 
+    if (!mudlet::self()->getAvailableFonts().contains(font, Qt::CaseInsensitive)) {
+        lua_pushnil(L);
+        lua_pushfstring(L, "Font '%s' is not available)", font.toUtf8().constData());
```

Comment:
```suggestion
        lua_pushfstring(L, "font '%s' is not available)", font.toUtf8().constData());
```
A small consistency fix